### ISSUE TITLE
strange thing when sampling

### ISSUE THREAD

**Issue by mayz20** [MinkaiXu/GeoDiff#15]
Hi, Minkai @MinkaiXu. Rencently I am trying to do some experiments based on GeoDiff, and having some problems.
Frist, I re-train the model with **your original hyper-parameters.** I notice that after around **800k** iterations, the loss hardly decreases but oscillates wildly. I don't know whether the model truly benefit from the training in the following **2m** iterations, and why the loss oscillates wildly?
Second, I am trying to sample some conformations from the checkpoints getting in my training procedure, but something very strange happened. Most of the molecules during sampling occur FloatingPointError and **retry with local clipping**. Then I use eval_covmat.py to evaluate the quailty of conformations, I get **0.00 Cov value and thousands of Mat value**. This strange thing happens all the time except one test with 700000.pt to generate molecules from 800 to 1000 (and I repeat the test again with same parameters and the strange thing occurs) I don't know why it happens because I run your code **without any modification**, and I wonder why you design the local clip operation and how to choose the corresponding hyper-parameter?
In the end, I would very appreciate it if you could give me some explanation and suggestions.

**Comment by MinkaiXu** [comment]
Hi, thanks for your interest!

1. This is typical for denoising score matching. See Fig.1 (right) in the original denoising score matching paper (https://arxiv.org/pdf/1907.05600.pdf). I'm not sure after 800K it can already generate good examples, but you may have a try.

2. Sorry I never met this before... Many people have run my code before, so I guess there might be some errors on your side.
